teos/src/config.rs

```diff
@@ -242,6 +243,147 @@ impl Config {
     pub fn is_default(&self) -> bool {
         self == &Config::default()
     }
+
+    /// checks non-default options using HashMap
+    pub fn get_non_default_options(&self) -> HashMap<String, (String, String)> {
```

Comment:
You still missed some Config options here. Check all the options in https://github.com/talaia-labs/rust-teos/blob/075b5047fac89169033cd903a0699c48b49963cb/teos/src/config.rs#L123.

Also, just so this is easier to follow, can you arrange them in the same order as they appear in the struct?

Reply:
> You still missed some Config options here. Check all the options in
> 
> https://github.com/talaia-labs/rust-teos/blob/075b5047fac89169033cd903a0699c48b49963cb/teos/src/config.rs#L123-L159
> 
> Also, just so this is easier to follow, can you arrange them in the same order as they appear in the struct?

Sure, I'll do it like that.